dali/ft/daftformat.cpp

```diff
@@ -58,6 +58,22 @@ CPartitioner::CPartitioner()
     partitioning = false;
 }
 
+void CPartitioner::setAbort(IAbortRequestCallback * _abort)
+{
+    abortChecker = _abort;
+}
+
+bool CPartitioner::isAborting()
+{
+    if (abortChecker && abortChecker->abortRequested())
+    {
+        LOG(MCdebugInfo, unknownJob, "CPartitioner::isAborting() -> Abort requested via callback");
```

Comment:
There can be e.g. 400 partitioners ? 
FileSpray already logs the abort, so 400 extra lines is excessive. 

Not a lot of point in the 'aborting' flag either, as it's a quick call into the FileSprayer once aborted,
so might as well be just:
```
return abortChecker && abortChecker->abortRequested();
```